flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -632,34 +643,32 @@ class TaskManager(
 
       case RefuseRegistration(reason) =>
         if (currentJobManager.isEmpty) {
-          log.error(s"The registration at JobManager $jobManagerAkkaURL was refused, " +
+          log.error(s"The registration at ResourceManager $jobManagerAkkaURL was refused, " +
```

Comment:
This is an artifact of refactoring the original approach that I followed where the TaskManager would register at the ResourceManager. Reverting.